pkg/oras/store.go

```diff
@@ -65,3 +67,54 @@ func (s *hybridStore) Writer(ctx context.Context, opts ...content.WriterOpt) (co
 	}
 	return s.ingester.Writer(ctx, opts...)
 }
+
+// TODO: implement (needed to create a content.Store)
+// TODO: do not return empty content.Info
+// Abort completely cancels the ingest operation targeted by ref.
+func (s *hybridStore) Info(ctx context.Context, dgst digest.Digest) (content.Info, error) {
```

Comment:
This is the only method that appears to be used by PushContents(). All tests are all passing, this object appears unused by oras at the moment..

Reply:
`containerd` introduced `annotateDistributionSourceHandler` in their code which requires `content.Manager`. Also, you are right that `Info()` is the only additional method used by `PushContents()`.

Reply:
From the `containerd` source code, the `annotateDistributionSourceHandler` tries to fetch the `Annotations` for the `Descriptor` from the  `content.Manager` interface. It is safe to return `content.Info{}` to bypass this process since `oras` has already taken care of the annotations in `oras.pack()`.